rules/anemic-vs-rich-domain-models/rule.md

```diff
@@ -0,0 +1,113 @@
+---
+seoDescription: Understand the differences between anemic and rich domain models to improve your software architecture.
+type: rule
+title: Do you understand the difference between anemic and rich domain models?
+uri: anemic-vs-rich-domain-models
+authors:
+  - title: Daniel Mackay
+    url: https://www.ssw.com.au/people/daniel-mackay
+created: 2024-06-17T13:42:55.753Z
+guid: 346475FF-7255-4BE2-BD09-C8113CA40399
+---
+
+When using Domain-Centric architectures such as Clean Architecture, we need to decide where the business logic will go. There are two main approaches to this: Anemic Domain Model and Rich Domain Model. Understanding the differences between these two models is crucial for making informed decisions about your software architecture.
+
+<!--endintro-->
+
+## Anemic Domain Model
+
+An Anemic Domain Model is characterized by:
+
+- **Property Bags:** Entities are simple data containers with public getters and setters.
+- **No Behavior:** No logic or validation within the entities.
+
+### Pros of Anemic Domain Model:
+
+- Good for simple or CRUD projects.
+- Easier to understand for developers new to the project.
+
+### Cons of Anemic Domain Model:
+
+- Doesn’t scale well with complexity.
```

Comment:
Needs qualification.

Reply:
+1 to this - I think we should clearly show or point out what part makes it "does not scale well" with "complexity"

Reply:
Have updated.  What do you think?